_co-op-companies/N-varian

```diff
@@ -0,0 +1,10 @@
+---
+title: "Varian"
+identification: "varian"
+description: "Varian uses intelligent approach to fight against cancer."
+link: "https://www.varian.com/"
+image: "assets/img/logos/varian.png"
+width: "200px"
+complete: false
```

Comment:
change complete to true for modal to show

Reply:
okay it is done